HW2/P3/P3.txt

```diff
@@ -0,0 +1,35 @@
+Problem 3:
+==========
+
+Below are some statistics from the runs in the 5 different versions.
+
+<<<<<
+
+Serial Version:
+---------------
+1074.656613 Million Complex FMAs in 4.27743196487 seconds,
+251.238738997 million Complex FMAs / second
+
+One Thread:
+-----------
+1074.656613 Million Complex FMAs in 4.24994087219 seconds,
+252.86389748 million Complex FMAs / second
+
+Two threads:
+------------
+1074.656613 Million Complex FMAs in 2.07837510109 seconds,
+517.065765673 million Complex FMAs / second
+
+Four threads:
+-------------
+1074.656613 Million Complex FMAs in 1.28325009346 seconds,
+837.449082199 million Complex FMAs / second
```

Comment:
Would have been useful to explicitly say that these were without instruction level parallelism